src/main/java/jewellery/inventory/model/Sale.java

```diff
@@ -18,8 +18,8 @@ public class Sale {
   @ManyToOne private User seller;
   @ManyToOne private User buyer;
 
-  @OneToMany(mappedBy = "partOfSale")
-  private List<Product> products;
+  @OneToMany(mappedBy = "sale", cascade = CascadeType.REMOVE)
```

Comment:
we need functionality to delete ProductPriceDiscount because we can have in one Sale more than one ProductPriceDiscount and we can return only one from all from a given Sale, but if we delete a Sale this guarantees us that all ProductPriceDiscounts to it will be deleted